pkg/subctl/cmd/validate_deployment.go

```diff
@@ -0,0 +1,133 @@
+/*
+© 2021 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	submarinerclientset "github.com/submariner-io/submariner-operator/pkg/client/clientset/versioned"
+	"github.com/submariner-io/submariner-operator/pkg/internal/cli"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+var validatePodsCmd = &cobra.Command{
+	Use:   "pods",
+	Short: "Validate the submariner pods",
+	Long:  "This command validates all the submariner pods are running",
+	Run:   validatePods,
+}
+
+func init() {
+	validateCmd.AddCommand(validatePodsCmd)
+}
+
+func validatePods(cmd *cobra.Command, args []string) {
+	configs, err := getMultipleRestConfigs(kubeConfig, kubeContext)
+	exitOnError("error getting REST config for cluster", err)
+
+	for _, item := range configs {
+		message := fmt.Sprintf("Validating Submariner Pods in %q", item.clusterName)
```

Comment:
We don't capitalize these in the help messages - either way we should be consistent.

```suggestion
		message := fmt.Sprintf("Validating submariner pods in %q", item.clusterName)
```

Reply:
Done.